xml_handlers/testing_handler.php

```diff
@@ -132,7 +132,7 @@ public function startElement($parser, $name, $attributes)
             $this->Label = new Label();
         } elseif ($name == 'TESTLIST' && $parent == 'TESTING') {
             $start_time = gmdate(FMT_DATETIME, $this->StartTimeStamp);
-            $end_time = gmdate(FMT_DATETIME, $this->EndTimeStamp); // The EndTimeStamp
+            // $end_time = gmdate(FMT_DATETIME, $this->EndTimeStamp); // The EndTimeStamp
```

Comment:
Interesting.  I'll take a closer look locally.  This looks like a bug to me, so if the tests are expecting the wrong answer we should update them.